components/LoginEntryPoint.tsx

```diff
@@ -24,6 +24,9 @@ const LoginEntryPoint = (props: Props) => {
       <button className={styles.entryButton} onClick={() => setLoginMethod(LoginMethod.EMAIL_WEBAUTHN)}>
         Email + WebAuthn (API Integration)
       </button>
+      <button className={styles.entryButton} onClick={() => setLoginMethod(LoginMethod.SDK_OAUTH)}>
+        OAuth + One Tap (SDK Integration)
+      </button>
```

Comment:
We'll need to clean up the login form, the height is no longer sufficient.

![image](https://user-images.githubusercontent.com/91094210/145288085-dad50996-6ae5-4438-9060-59393b7baf7d.png)

Reply:
https://github.com/stytchauth/stytch-nextjs-integration/pull/47/commits/56ff7fc6a584a818e56680342278b45e96bbb430